src/com/lftechnology/java/training/dipak/employeemanagement/dao/EmployeeDao.java

```diff
@@ -0,0 +1,200 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.dipak.employeemanagement.api.EmployeeApi;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.Employee;
+
+import com.mysql.jdbc.exceptions.MySQLIntegrityConstraintViolationException;
+
+/**
+ * <p>This class implements the EmployeeApi and has definition to all the
+ * overwritten methods.</p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class EmployeeDao implements EmployeeApi {
+
+	private final String CONNECTION_FAILED_MESSAGE = "Connection Setup Failed";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "CONNECTION_FAILED_MESSAGE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)